tests/row_generation/test_MethodPlaceNotationGenerator.py

```diff
@@ -56,5 +58,113 @@ def test_parse_block(self):
         self.assertEqual(7, stage)
 
 
+class SpecialMethodNameTests(TestCase):
+    def test_grandsire__odd_stages(self):
+        test_cases = [
+            ("Grandsire Doubles", 5),
+            ("GrandSire DouBles", 5),
+            ("grandsire doubles", 5),
+            ("GRANDSIRE DOUBLES", 5),
+            ("  Grandsire   Doubles  ", 5),
+            ("Grandsire Triples", 7),
+            ("Grandsire Caters", 9),
+            ("Grandsire Cinques", 11),
+        ]
+
+        for (method_title, expected_stage) in test_cases:
+            with self.subTest(method_title=method_title, expected_stage=expected_stage):
+                generator = generator_from_special_title(method_title)
+                self.assertIsInstance(generator, PlaceNotationGenerator)
+                self.assertTrue(expected_stage, generator.stage)
+
+    def test_grandsire__other_stages(self):
+        test_cases = [
+            ("Grandsire Singles"),
+            ("Grandsire Minimus"),
+            ("Grandsire Minor"),
+            ("Grandsire Major"),
+            ("Grandsire Royal"),
+            ("Grandsire Maximus"),
```

Comment:
Nice to see that even bell Grandsire is supported now! :)

Reply:
It is now